readme.md

```diff
@@ -53,6 +53,14 @@ Type: `boolean`<br>
 Default: `true`
 
 Automatically stop checking for the element to be ready after the [DOM ready event](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event). The promise is then resolved to `undefined`.
+The element that's expected to contain a match.
+
+##### timeout
+
+Type: `number | false`<br>
+Default: `false`
+
+The time to wait before stopping the search, in milliseconds.
```

Comment:
```suggestion
Time in milliseconds to wait before stopping the search and resolving the promise to `undefined`.
```

?

Reply:
`Time in milliseconds to wait` sounded awkward, maybe it's ok.

How about `Milliseconds to wait`?

Reply:
> How about Milliseconds to wait?

👍